k-distribution/include/builtin/domains.k

```diff
@@ -586,21 +598,22 @@ module STRING-BUFFER-SYNTAX
   syntax String ::= StringBuffer2String ( StringBuffer ) [function, functional, hook(BUFFER.toString)]
 endmodule
 
-module STRING-BUFFER-SYMBOLIC [symbolic]
-  imports STRING-BUFFER-SYNTAX
-  imports STRING-SYNTAX
-  imports BOOL-SYNTAX
-
-  syntax StringBuffer ::= String
+module STRING-BUFFER-CONCRETE [concrete, kast]
+  imports STRING-BUFFER-HOOKED
+endmodule
 
-  rule .StringBuffer => ""
+module STRING-BUFFER-SYMBOLIC [symbolic]
+  imports STRING-BUFFER-IN-K
+endmodule
 
-  rule StringBuffer2String(S:String) => S
+module STRING-BUFFER-KORE [kore]
```

Comment:
this isn't actually what we want. We want to use the hooks for the ocaml and llvm backends, not just the ocaml backend. So you should delete this module and the kast attribute above.

Reply:
@dwightguth Does this seem right?